wdl-lint/src/rules/meta_sections.rs

```diff
@@ -91,18 +91,18 @@ fn missing_sections(name: Ident, context: Context) -> Diagnostic {
 
 /// A lint rule for missing meta and parameter_meta sections.
 #[derive(Default, Debug, Clone, Copy)]
-pub struct MissingMetasRule {
+pub struct MetaSectionsRule {
     /// The version of the WDL document being linted.
     version: Option<SupportedVersion>,
 }
 
-impl Rule for MissingMetasRule {
+impl Rule for MetaSectionsRule {
     fn id(&self) -> &'static str {
         ID
     }
 
     fn description(&self) -> &'static str {
-        "Ensures that tasks have both a meta and a parameter_meta section."
+        "Ensures that tasks and workflows have the required `meta` and `parameter_meta` sections."
```

Comment:
They are applied to workflows too, please correct me if im wrong

Reply:
correct! Nice catch